mlflow/gateway/providers/openai.py

```diff
@@ -34,8 +88,8 @@ async def chat(self, payload: chat.RequestPayload) -> chat.ResponsePayload:
         payload["temperature"] = 2 * payload["temperature"]
 
         resp = await send_request(
-            headers=self.headers,
-            base_url=self.base_url,
+            headers=self._request_headers,
+            base_url=self._request_base_url,
             path="chat/completions",
             payload={"model": self.config.model.name, **payload},
```

Comment:
Tested and it appears to be fine - Azure ignores it

Reply:
I'll remove it anyway :)

Reply:
Removed!

Reply:
Can we add a comment?

Reply:
Can we make the `model.name` field optional if Azure OpenAI doesn't need it?

Reply:
I don't think we should make it optional, since the "model" field gets returned to the user when they do a GetRoute / SearchRoutes, and it's meant to tell the user what kind of model the route is serving.

Reply:
Added inline comments :)